### DISCUSSION TITLE
Deepin unioncode cannot run properly

### DISCUSSION THREAD

**Discussion by mukif2** [linuxdeepin/developer-center#7069] [Development Questions | 开发答疑]
Hello, I'v successfully built deepin unioncode on version 1.19.0. However, when I tried to run it, something went wrong:
![Screenshot at 2024-01-24 21-20-36](https://github.com/linuxdeepin/developer-center/assets/40907652/defe0a2c-1e98-4737-a091-e65f52d96149)

I'm working on Loongnix 20.5 with Loongson3A6000;
dtkcore/widget/gui built from source code, on version 5.6.16;
QT5.15.2 from loongson apt repo

**Comment by BLumia** [comment]
cc @deepin-mozart @Kakueeen

**Comment by deepin-mozart** [comment]
> Hello, I'v successfully built deepin unioncode on version 1.19.0. However, when I tried to run it, something went wrong: ![Screenshot at 2024-01-24 21-20-36](https://private-user-images.githubusercontent.com/40907652/299327545-defe0a2c-1e98-4737-a091-e65f52d96149.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDYxNzYzNzgsIm5iZiI6MTcwNjE3NjA3OCwicGF0aCI6Ii80MDkwNzY1Mi8yOTkzMjc1NDUtZGVmZTBhMmMtMWU5OC00NzM3LWEwOTEtZTY1ZjUyZDk2MTQ5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTI1VDA5NDc1OFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTJhNTViNjk4MTg5ODZhZTlhNmYzZjA0YWZmMzJiOTc2Zjg0NzE4OGZlMmZhYTRhMGVhZDhiNmJiMDFlMzJhMDkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Sv6aSzBaNt9Po7VUg1a4MM86fLjLVZ1X4v1Bz7IsG7E)
> 
> I'm working on Loongnix 20.5 with Loongson3A6000; dtkcore/widget/gui built from source code, on version 5.6.16; QT5.15.2 from loongson apt repo

抱歉，我们没有对应的环境复现问题，不过你可以尝试以下操作：
![image](https://github.com/linuxdeepin/developer-center/assets/33650326/4d8e2cd1-8795-45b9-822b-65d9f8062e60)
